src/main/scala/com/azavea/modellab/Service.scala

```diff
@@ -60,11 +61,13 @@ object Service extends SimpleRoutingApp with DataHubCatalog  with App {
 
   def registerLayerRoute = post {
     requestInstance { req =>
-      complete {
-        val json = req.entity.asString.parseJson
-        val node = parser.parse(json)
-        println(s"Registered: $node")        
-        StatusCodes.Accepted
+      respondWithHeader(RawHeader("Access-Control-Allow-Origin", "*")) {
```

Comment:
I think we'd just put a web server like Nginx in front of this service, so specifics like CORS don't have to live within the application.